jenkins_jobs/modules/scm.py

```diff
@@ -1679,8 +1679,14 @@ class SCM(jenkins_jobs.modules.base.Base):
     def gen_xml(self, xml_parent, data):
 
         # multibranch-pipeline scm implementation is incompatible with SCM
-        if data.get("project-type") in ["multibranch", "multibranch-defaults"]:
-            logging.debug("SCM Module skipped for multibranch project-type.")
+        if data.get("project-type") in [
+            "multibranch",
+            "multibranch-defaults",
+            "pipeline",
```

Comment:
Forgotten comma after last list element  ?

Reply:
Interestingly the comma, and the whole formatting comes from [black](https://github.com/psf/black), which is automatically run by tox.